sci-visualization/fityk/fityk-1.3.1.ebuild

```diff
@@ -0,0 +1,93 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 python3_4 )
+
+inherit fdo-mime python-r1 wxwidgets
+
+DESCRIPTION="General-purpose nonlinear curve fitting and data analysis"
+HOMEPAGE="http://fityk.nieto.pl/"
+SRC_URI="https://github.com/wojdyr/${PN}/releases/download/v${PV}/${P}.tar.bz2"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="gnuplot nlopt readline python static-libs wxwidgets"
+
+REQUIRED_USE="python? ( ${PYTHON_REQUIRED_USE} )"
+
+CDEPEND="
+	>=sci-libs/xylib-1
+	>=dev-lang/lua-5.1:0
+	nlopt? ( sci-libs/nlopt )
+	python? ( ${PYTHON_DEPS} )
+	readline? ( sys-libs/readline:* )
```

Comment:
if it links to readline, this needs to be `:0=`, it doesnt link, it needs to be `:0`